xrdsst/controllers/endpoint.py

```diff
@@ -201,3 +282,40 @@ def get_services_description(self, config):
                                    'client': client,
                                    'security_server': security_server,
                                    'ss_api_config': ss_api_config}
+
+    def remote_list_endpoints(self, ss_api_config, ss_name, service_description_ids):
+        service_descriptions_api = ServiceDescriptionsApi(ApiClient(ss_api_config))
+        endpoints_list = []
+        render_data = []
+        for service_description_id in service_description_ids:
+            try:
+                service_description = service_descriptions_api.get_service_description(service_description_id)
+                endpoints_list.extend(self.parse_service_description_to_endpoint_table(service_description))
```

Comment:
Maybe we should not display endpoints for WSDLs and show some message instead that endpoints are displayed only for REST and OPENAPI3?